OmniKitUI/ViewControllers/ReplacePodViewController.swift

```diff
@@ -15,37 +15,50 @@ class ReplacePodViewController: SetupTableViewController {
 
     enum PodReplacementReason {
         case normal
-        case fault
+        case fault(_ eventType: FaultEventCode.FaultEventType)
```

Comment:
If you make the fault case take a PodInfoFaultEvent, then you can handle the unknown types as well.